.github/workflows/publish.yml

```diff
@@ -0,0 +1,15 @@
+name: Publish
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch: {}
+
+jobs:
+  publish:
+    uses: wetransform/gha-workflows/.github/workflows/sbt-library-check.yml@master
+    with:
+      java-version: 17
+      expect-tests: false # library does not contain any tests yet
+    secrets: inherit
```

Comment:
`inherit` does not work here because the shared workflow is from another organization, so secrets need to be passed on individually.
Required secrets are listed here:
https://github.com/stempler/gha-workflows/blob/0ff5d1c39bbb1f1588778c74d993573072752294/.github/workflows/sbt-library-publish.yml#L21